model_compression_toolkit/core/tpc_models/qnnpack/target_platform_capabilities.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import importlib
+
+from model_compression_toolkit.core.common.constants import TENSORFLOW, PYTORCH
+from model_compression_toolkit.core.common.target_platform import TargetPlatformCapabilities
+
+###############################
+# Build Tensorflow TPC models
+###############################
+
+found_tf = importlib.util.find_spec("tensorflow") is not None and importlib.util.find_spec(
```

Comment:
This code repeat itself too many time, please make it a function and define FOUND_TF and FOUND_TORCH as constant in common.

Reply:
done